inputs_postgres12.yml

```diff
@@ -0,0 +1,55 @@
+---

```

Comment:
Rename this to inputs_postgres12_example.yml. And make sure to commit the deleted file inputs.yml. Then it will show that this file was simply renamed.

Reply:
Renamed and added the deleted files and re-commited